src/heap/src/AivHeap.c

```diff
@@ -6,11 +6,11 @@
 #include "Include_i.h"
 
 #ifdef HEAP_DEBUG
-    AIV_ALLOCATION_HEADER gAivHeader = {{0, AIV_ALLOCATION_TYPE, 0, ALLOCATION_HEADER_MAGIC}, 0, ALLOCATION_FLAGS_FREE, NULL, NULL};
-    AIV_ALLOCATION_FOOTER gAivFooter = {{0, ALLOCATION_FOOTER_MAGIC}};
+    struct AIV_ALLOCATION_HEADER gAivHeader = {{0, AIV_ALLOCATION_TYPE, ALLOCATION_FLAGS_FREE, ALLOCATION_HEADER_MAGIC}, 0, NULL, NULL};
```

Comment:
Just a thought, but, can we define the values for every field using the . operator? It will just be safer to do so, in case somebody changes the order of structure items in the future?

Reply:
These are static definitions so you can't dynamically execute code. Moreover, this is C and not C++. I would imagine anyone changing a piece of code at this layer would be super familiar with what's going on.